src/cljs/vanilla/widgets/sankey_chart.cljs

```diff
@@ -1,7 +1,11 @@
 (ns vanilla.widgets.sankey-chart
-  (:require [vanilla.widgets.make-chart :as mc]))
+  (:require [vanilla.widgets.make-chart :as mc]
+            ["react-highcharts" :as ReactHighcharts]
+            ["highcharts/modules/sankey" :as addSankeyModule]))
 
 
+(addSankeyModule ReactHighcharts/Highcharts)
```

Comment:
This seems out of place

Reply:
this one I've already tried inside core.cljs and it does NOT work there. I don't know why, but the sankey only resolves when this call is right here.

Reply:
done. see https://github.com/cawasser/vanilla/pull/49/commits/2cc489cb61f41bdd7c44ea2633646579d56a61e3